decoder-js/src/main/java/com/hpe/caf/decoder/PropertyRetriever.java

```diff
@@ -27,4 +37,23 @@ public class PropertyRetriever {
     public static String getenv(String key){
         return System.getenv(key);
     }
+
+    /**
+     * Gets the contents of the file pointed to by the specified environment variable.
+     * @param key Name of environment variable containing the path of the file to read from.
+     * @return Contents of the file pointed to by the specified environment variable, or null if the specified environment variable is
+     * null or empty.
+     * @throws IOException If the file could not be read for any reason.
+     */
+    public static String getenvfile(String key) throws IOException {
```

Comment:
A better name here might be `getFileContentFromEnv` but I was trying to stay consistent with the short, lowercase format of the existing `getenv` method.